### ISSUE TITLE
openssl 3.1.1 cannot run normally after compiling, including the rpm package downloaded from github

### ISSUE THREAD

**Issue by ssdsl0126** [philyuchkoff/openssl-RPM-Builder#11]
![image](https://github.com/philyuchkoff/openssl-RPM-Builder/assets/35905829/6b49b142-6ef3-41f2-882e-d13ae16e3989)

**Comment by philyuchkoff** [comment]
Check [here](https://github.com/openssl/openssl/issues/18025), please.

**Comment by dmitry-j-mikhin** [comment]
Problem still exist and can be reproduced by following command:
```
$ docker run -it --rm centos:7 sh -c "rpm -i --nodeps https://github.com/philyuchkoff/openssl-RPM-Builder/releases/download/3.1.2/openssl-3.1.2-1.el7.x86_64.rpm && openssl version"
openssl: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
```
Wrong links were used in openssl3 package. They were fixed in #13 PR, new package can be tested with following command:
```
$ docker run -it --rm centos:7 sh -c "rpm -i --nodeps https://github.com/dmitry-j-mikhin/openssl-RPM-Builder/releases/download/3.1.2/openssl-3.1.2-1.el7.x86_64.rpm && openssl version"
OpenSSL 3.1.2 1 Aug 2023 (Library: OpenSSL 3.1.2 1 Aug 2023)
```
